# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [2]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [37]:
rand1 = np.random.randint(1, high = 5, size = 100)
rand2 = np.random.randint(1, high = 4, size = 100)
rand3 = np.random.randint(10000, high = 30001, size = 100)

### Step 3. Let's create a DataFrame by joinning the Series by column

In [13]:
df = pd.DataFrame(list(zip(rand1, rand2, rand3)))

In [14]:
df.head()

0  1      2
0  4  2  12345
1  2  2  27799
2  3  1  29652
3  4  1  23750
4  1  3  18931

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [15]:
df.columns = ['bedrs', 'bathrs', 'price_sqr_meter']

In [16]:
df.head()

bedrs  bathrs  price_sqr_meter
0      4       2            12345
1      2       2            27799
2      3       1            29652
3      4       1            23750
4      1       3            18931

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [38]:
rand1 = list(rand1)
rand2 = list(rand2)
rand3 = list(rand3)

In [39]:
rand1.extend(rand2)

In [40]:
rand1.extend(rand3)

In [41]:
df2 = pd.DataFrame(rand1)

In [42]:
df2

0
0        4
1        4
2        3
3        2
4        2
..     ...
295  19424
296  14793
297  18443
298  27375
299  28931

[300 rows x 1 columns]

### Step 6. Oops, it seems it is going only until index 99. Is it true?

### Step 7. Reindex the DataFrame so it goes from 0 to 299